addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,13 @@
-.o_dialog {
-  > .modal {
-    display: block;
-  }
-}
-
-.modal.o_technical_modal {
-  .modal-content {
-    border-radius: 0;
-
-    .modal-header .modal-title {
-      word-break: break-word;
-    }
-
-    .modal-footer {
-      flex-wrap: wrap;
-      text-align: left;
-      justify-content: flex-start;
-
-      footer {
-        > :not(:first-child) {
-          margin-left: 0.25rem;
-        }
-        > :not(:last-child) {
-          margin-right: 0.25rem;
+.modal.o_technical_modal .modal-footer {
+    button {
+        margin: map-get($spacers, 0) map-get($spacers, 1) map-get($spacers, 1);
+  
+        &:last-child {
+          margin-right: map-get($spacers, 0);
         }
-        button {
-          margin-bottom: 0.5rem;
+  
+        &:first-child {
+          margin-left: map-get($spacers, 0);
         }
-      }
-    }
-  }
-
-  @include media-breakpoint-up(sm) {
-    .modal-dialog .modal-content.o_dialog_error .modal-body {
-      overflow: visible;
-      display: flex;
-      flex-flow: column nowrap;
-
-      > .alert,
-      > button {
-        flex: 0 0 auto;
-      }
-
-      > .o_error_detail {
-        flex: 1 1 auto;
-        min-height: 0;
-        overflow: auto;
-      }
     }
   }
```

Comment:
General indentation issues... this last bracket is not the first character of the line.
Also, always use 4 spaces for indentation https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines#generic-rules